test/Swap.test.ts

```diff
@@ -221,5 +221,13 @@ describe("Swap RBTC", function () {
     expect(senderBalance).to.be.equal(oneEther);
 
     await expect(anotherSideToken.connect(sender).send(swapRBTC.address, halfEther, "0x")).to.be.revertedWith("SwapRBTC: Side Token not found");
-41  });
+  });
+
+  it('Should validate an invalid address when call the token received method', async function () {
+    await expect(swapRBTC.tokensReceived(bnbContract, sender.address, deployer.address, 0, 0, 0)).to.be.revertedWith("SwapRBTC: Invalid 'to' address");
```

Comment:
```suggestion
    await expect(swapRBTC.tokensReceived(bnbContract, sender.address, deployer.address, 0, 0, 0)).to.be.revertedWith("SwapRBTC: Invalid 'to' address");
```

Test is correct but you should use more descriptive infromation on the example. The first parameter here is bnbContract, but this should represent the operator https://eips.ethereum.org/EIPS/eip-777#erc777tokensrecipient-and-the-tokensreceived-hook
As we are not using any kind of test over the operator this value could be anything, but is better to use address zero, or const operator = < a random address>. so the people that are reading the test understand that this value is not important. 
I also recomend to use the same parameters as the call that is successful so it's clearer the parameter that it was changed in order to make it fail. That being said please add a test for the tokensReceived successful operation, only adding the failing paths it's not recommended